python_modules/libraries/dagster-dbt/dagster_dbt/include/dbt_scaffold/definitions.py

```diff
@@ -0,0 +1,32 @@
+from pathlib import Path
+
+from dagster import Definitions
+
+from dagster_dbt import DbtCli, DbtManifest, dbt_assets
+
+DBT_PROJECT_DIR = Path(__file__).parent.joinpath("..", "jaffle_shop")
+DBT_MANIFEST_PATH = DBT_PROJECT_DIR.joinpath("target", "manifest.json")
+DBT_MANIFEST = DbtManifest.read(path=DBT_MANIFEST_PATH)
+
+
+@dbt_assets(manifest=DBT_MANIFEST)
+def my_dbt_assets(dbt: DbtCli):
```

Comment:
Is `my_dbt_assets` the name that we would expect people to use in a "realistic" project setup? If not, is there a different one?

Reply:
The name of the decorated function here corresponds to the underlying op definition. I assume that we're trying to name that accurately.

From the user's perspective, this corresponds to [running a dbt project](https://docs.getdbt.com/docs/running-a-dbt-project/run-your-dbt-projects). So perhaps:

- `build_dbt_project`
- `run_dbt_project`

Reply:
For `@asset`- and `@multi_asset`- decorated functions, the name applies to the op, but it also the variable name of the `AssetsDefinition` object.  So our general convention is for the name to be a name that's appropriate for a reference to that object, i.e. a noun that describes that collection of assets.

I think it's worth following that convention here.  It does result in a weird name for the op, but no weirder than for other non-dbt ops that are used in similar contexts.

Reply:
We could use the name of the dbt project in which we are scaffolding. We can get this name from `dbt_project.yml`. For example, the jaffle shop project has the name `jaffle_shop`.

Reply:
That makes sense to me. Something like `all_dbt_assets` could be fine too if easier.